plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Default.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# OPTIONS_GHC -O2 #-}
+
+-- | This module defines a 'ToBuiltinsRuntime' instance for 'CekValue'. It must be the only thing
+-- that this module defines. The idea is that dumping the Core of this module gives you the whole
```

Comment:
> It must be the only thing that this module defines.

Maybe a word or two in capital letters here would be useful just to make sure no-one misses it.

Reply:
I made it "MUST", but I think it kinda deflates the value of MUST. It's not a huge deal if someone adds some definition to this module for some reason, it's just not recommended to do so in order for Core to stay digestible. Anyway, done.